README.md

```diff
@@ -1,3 +1,7 @@
+![seminar](./Seminar.png)
+
+
+
```

Comment:
These lines are extra - can you remove them so there is just one empty line between the image and heading?